check_process

```diff
@@ -16,17 +16,19 @@
 		setup_private=1
 		setup_public=1
 		upgrade=1
-		backup_restore=0
-		multi_instance=0
+		upgrade=1		from_commit=cc5164cdadf9aec0eebb25d4252d849aa0054606
+		upgrade=1		from_commit=65cd8b542310e8b349107ed222fdc57e3c2d239a
+		backup_restore=1
+		multi_instance=1
 		incorrect_path=1
 		port_already_use=0
 		change_url=0
 ;;; Levels
 	Level 1=auto
 	Level 2=auto
 	Level 3=auto
-# Level 4:
-	Level 4=0
+# Level 4: the tool can't talk with LDAP
+	Level 4=1
```

Comment:
Note that this whole level thing changed and Level 4 doesn't mean this anymore. Now only level 5 means anything for this file, c.f. https://github.com/YunoHost/example_ynh/blob/master/check_process.default#L32 for example

Reply:
Ok, I didn't know, where can I read more info about this?